tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -185,7 +185,7 @@ sub upload_virt_logs {
     my ($log_dir, $compressed_log_name) = @_;
 
     my $full_compressed_log_name = "/tmp/$compressed_log_name.tar";
-    assert_script_run("tar cvf $full_compressed_log_name $log_dir; gzip -f $full_compressed_log_name; rm $log_dir -r", 60);
+    script_run("tar cvf $full_compressed_log_name $log_dir; gzip -f $full_compressed_log_name; rm $log_dir -r", 60);
```

Comment:
Don't we want to log some info in case this command fails so it's visible in the dashboard? Other than that LGTM

Reply:
Thanks for the review.
Actually we use loose check here because for some tests, it will return non-zero because some files in the upload log file list do not exist for some specific test result, although for most cases, it should exist and pass So this fail or pass result is not so meaningful.
And we have save_screenshot below this sentence, so  if it fails,  it is visible and debuggable already.